src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -42,8 +42,8 @@ public class CommandTestUtil {
     public static final String VALID_FAMILY_AMY = "4";
     public static final String VALID_FAMILY_BOB = "4";
 
-    public static final String VALID_TAG_HUSBAND = "husband";
-    public static final String VALID_TAG_FRIEND = "friend";
+    public static final String VALID_TAG_AMY = "buyer";
+    public static final String VALID_TAG_BOB = "seller";
```

Comment:
hm maybe can consider renaming this to its actual tag for easier understanding in other test files as well
```suggestion
    public static final String VALID_TAG_BUYER = "buyer";
    public static final String VALID_TAG_SELLER = "seller";
```

Reply:
Sorry but it is to create new arbitrary persons with arbitrary tags so do leave it as it is